### ISSUE TITLE
Match Height Of Hidden Elements

### ISSUE THREAD

**Issue by CrispySinger** [liabru/jquery-match-height#160]
Hi, 

I checked the existing issues, but couldn't see one that matched what I'm looking for. I may be incorrect, sorry if so. 

I'm aiming to use match height within a mega-menu navigation, due to the children items being set to display until toggle / hover, the height isn't grabbed. I tried resorting to changing opacity values over display none but that may be more problematic down the line. 

I'm just wondering if there is already a solution for getting the height of hidden elements already, or if I may be required to put something in place to tackle this issue. 

Cheers, 
Jordan Smith

**Comment by liabru** [comment]
Yes there is code to handle matching heights of hidden elements, though there may be edge cases where it could fail. My suggestion is to ensure visibility of your elements before matching then hiding them, if possible. Otherwise you'll need to supply a link demonstrating the issue to investigate.
